Libraries/LibJS/Runtime/Exception.h

```diff
@@ -37,12 +37,14 @@ class Exception : public Cell {
     virtual ~Exception() override;
 
     Value value() const { return m_value; }
+    Vector<String> trace() const { return m_trace; }
```

Comment:
Let's return this as a `const Vector<String>&` to avoid copying the `Vector` every time.